src/Tmds.ExecFunction/ExecFunction.cs

```diff
@@ -224,7 +226,28 @@ static ExecFunction()
                 }
 
                 HostArguments = PasteArguments.Paste(new string[] { "exec", "--runtimeconfig", runtimeConfigFile, "--depsfile", depsFile, execFunctionAssembly });
-            }
+                if(isTestHost)
```

Comment:
Can you move this up, between` isTestHost` assignment and `EndsWith` `dotnet` check?

Reply:
Moved.